_sass/variables.scss

```diff
@@ -1,8 +1,27 @@
 $herobackground:  #1c1e26;
 $hero_font:  'Space Mono', monospace;
-$font_family: "Open Sans", sans-serif;
+
 $var_color: #F87669;
 $var_name_color:  #0ea87f;
 $var-value_color: #fcb350;
 $title_color: white;
 $wrap_color: #EED0D4;
+
+
```

Comment:
since these are all variables we can just leave a single line of space for different groups